File: app/src/main/java/in/ecomexpress/sathi/repo/remote/model/rts/Response.java
package in.ecomexpress.sathi.repo.remote.model.rts;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Response {

    @JsonProperty("description")
    public String description;
    @JsonProperty("status-code")
    public String status_code;

    @JsonProperty("code")
    public String code;
}

File: app/src/main/java/in/ecomexpress/sathi/repo/remote/model/forward/reassign/Response.kt
package `in`.ecomexpress.sathi.repo.remote.model.forward.reassign

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
data class Response(
    val description: String,
    val is_dc_update_allowed_for_dept: Boolean,
)
